helm_deploy/laa-court-data-adaptor/templates/_environment.tpl

```diff
@@ -77,7 +89,10 @@ env:
   - name: AWS_REGION
     value: {{ .Values.sqs_region }}
   - name: SENTRY_DSN
-    value: {{ .Values.sentry_dsn }}
+    valueFrom:
+      secretKeyRef:
+        name: aws-secrets
+        key: sentry_dsn
```

Comment:
Are we using the same sentry dsn for all env? Also, I see prod and production Envs in Sentry https://ministryofjustice.sentry.io/projects/laa-court-data-adaptor/?project=5375870
Should we cleanup the envs not used?

Reply:
Yes, it looks like CDA is currently using the same DSN for all envs. I haven't changed that, but maybe we should separate it all out.

Reply:
Happy to leave it as it is. Having one DSN with different environments is probably easier to maintain. But, I see that laa-court-data-api has env specific DSN. May be we can create separate story to group the - laa-court-data-api  DSNs to one.